### ISSUE TITLE
NullPointerException when the definitionPointer is not correct

### ISSUE THREAD

**Issue by slecholineas** [bjansen/swagger-schema-validator#9]
When specifying a definitionPointer which does not exist in the swagger, a NPE is generated. It would be nice if this would result in an "Incorrect definitionPointer" error message :)

`java.lang.NullPointerException
	at com.github.fge.jsonschema.main.JsonValidator.buildJsonSchema(JsonValidator.java:188)
	at com.github.fge.jsonschema.main.JsonSchemaFactory.getJsonSchema(JsonSchemaFactory.java:205)
	at com.github.bjansen.ssv.SwaggerValidator.getSchema(SwaggerValidator.java:131)
	at com.github.bjansen.ssv.SwaggerValidator.validate(SwaggerValidator.java:94)
	at com.github.bjansen.ssv.SwaggerValidator.validate(SwaggerValidator.java:81)`

**Comment by bjansen** [comment]
Yep, totally. Thanks for reporting the problem!

**Comment by abergmeier** [comment]
It seems like we get the same error when trying to validate `{}`.

**Comment by bjansen** [comment]
Actually I get this exception instead: 
```
com.github.fge.jsonschema.core.exceptions.JsonReferenceException: fatal: danglingRef
    level: "fatal"
```

It looks like the recent upgrade of `json-schema-validator` changed that NPE to the same exception as in #11

**Comment by bjansen** [comment]
I just published version 0.4.1 which provides a better error message
